### PR TITLE
[DEV-4549] team origin node id column

### PR DESCRIPTION
**Pull request by gaetano-guerriero**

### CODE REVIEW

File: server/scripts/backfill_teams_meta_id.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+    await sdb.connect()
+    await mdb.connect()
+
+    print(args.account)
+    await _backfill_multiple_accounts(args.overwrite, args.account, sdb, mdb)
+
+
+def _get_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser()
```
Comment (vmarkovtsev) [inline server/scripts/backfill_teams_meta_id.py:38]:
```python
flogging.add_logging_args(parser)
```
and delete the manual `logging.basicConfig`

File: server/scripts/backfill_teams_meta_id.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+async def _main() -> None:
+    args = _get_args()
+    logging.basicConfig(level=logging.INFO)
+
+    sdb = measure_db_overhead_and_retry(Database(args.state_db))
+    mdb = measure_db_overhead_and_retry(Database(args.metadata_db))
+
+    await sdb.connect()
+    await mdb.connect()
```
Comment (vmarkovtsev) [inline server/scripts/backfill_teams_meta_id.py:31]: Can be `gather()`-ed

**Review by vmarkovtsev — CHANGES_REQUESTED** [review]

File: server/scripts/backfill_teams_meta_id.py (hunk 2 above)
Comment (gaetano-guerriero) [reply server/scripts/backfill_teams_meta_id.py:31]: sure, note this file will be trashed

File: server/athenian/api/models/state/models.py [hunk 3]
```diff
@@ -257,7 +267,7 @@ class Feature(create_time_mixin(updated_at=True), Base):
     )
 
 
-class AccountFeature(create_time_mixin(updated_at=True), Base):
+class AccountFeature(create_time_mixin(created_at=True, updated_at=True), Base):
```
[the remark this answers is not in the document]
Comment (gaetano-guerriero) [reply server/athenian/api/models/state/models.py:270]: Do you mean we don't need it but the migration was forgotten, or we don't need and we cannot migrate ?

File: server/athenian/api/internal/account.py [hunk 4]
```diff
@@ -377,33 +394,31 @@ async def get_account_organizations(
...
-        select([func.count(StateTeam.id)]).where(
-            and_(
-                StateTeam.owner_id == account,
-                StateTeam.name != StateTeam.BOTS,
-                StateTeam.id != root_team_id,
-            ),
+        select(func.count(StateTeam.id)).where(
+            StateTeam.owner_id == account,
+            StateTeam.name != StateTeam.BOTS,
+            StateTeam.id != root_team_id,
```
[the remark this answers is not in the document]
Comment (gaetano-guerriero) [reply server/athenian/api/internal/account.py:413]: Cannot find in changelog, but yes in 1.3 it was `where(whereclause)` not it's `where(*whereclause)`

File: server/athenian/api/internal/account.py [hunk 5]
```diff
@@ -107,6 +108,29 @@ async def get_metadata_account_ids_or_empty(
         return ()
 
 
+@cached(
+    exptime=middle_term_exptime,
+    serialize=pickle.dumps,
+    deserialize=pickle.loads,
+    key=lambda account, **_: (sorted(account),),
```
Comment (vmarkovtsev) [inline server/athenian/api/internal/account.py:115]: Whoopsie, the cache key doesn't match the argument name.

**Review by vmarkovtsev — APPROVED** [review]
Please merge once the cache key is fixed :+1:

**Pull request merged**